src/main/java/pl/marcin/project/runners/WebAppRunner.java

```diff
@@ -20,18 +24,32 @@ public class WebAppRunner implements AppRunner {
     public void runApplication() {
         ConfigurableApplicationContext context = SpringApplication.run(ProjectApplication.class);
         var customerService = context.getBean(CustomerEntityService.class);
+        var mapService = context.getBean(GeoSerwis.class);
+        try {
+            findLocation(mapService);
+        } catch (Exception e) {
+            throw new RuntimeException(e);
+        }
         createCustomer(customerService);
 //        updateCustomer(customerService, 5L);
         getAllCustomers(customerService);
 //        getCustomerById(customerService, 5L);
-        deleteCustomerWithId(customerService, 6L);
+//        deleteCustomerWithId(customerService, 6L);
 
         var cupService = context.getBean(CupEntityService.class);
         createCup(cupService);
-        updateCup(cupService, 1L);
+//        updateCup(cupService, 1L);
         getAllCup(cupService);
-        getCupById(cupService, 1L);
-        deleteCupWithId(cupService, 1L);
+//        getCupById(cupService, 1L);
+//        deleteCupWithId(cupService, 1L);
+    }
+
+    private void findLocation(GeoSerwis mapService) throws Exception {
+        AddressData addressDataStart = new AddressData("34-300", "Żywiec", "Baczyńskiego", 12);
```

Comment:
I hope it's not your home address ;) this repo is public. It's better to keep test data anonymous